components/LearnLayout.tsx

```diff
@@ -11,6 +15,20 @@ function LearnLayout(props: PropsWithChildren<{}>) {
         </p>
       </aside>
       <div className="w-full md:pl-72">
+      <div className="mb-12">
+        <Link href="/learn/foundations/about-nextjs" passHref>
+          <button 
+          className={`px-3.5 py-2 ${currentPath === "about-nextjs" ?`text-white bg-[#0070f3] shadow-[rgba(0,118,255,.39)] shadow-lg` : "text-[#0070f3]"} mr-1 text-center transition-all rounded-lg cursor-pointer hover:opacity-90`}>
+            1
+          </button>
+        </Link>
+        <Link href="/learn/foundations/about-nextjs/what-is-nextjs" passHref>
+          <button 
+          className={`px-3.5 py-2 ${currentPath === "what-is-nextjs" ?`text-white bg-[#0070f3] shadow-[rgba(0,118,255,.39)] shadow-lg` : "text-[#0070f3]"} mr-1 text-center transition-all rounded-lg cursor-pointer hover:opacity-90`}>
+            2
+          </button>
+        </Link>
```

Comment:
This component `<LearnLayout />` is not only applied to the `/learn/foundations` but also will be applied to all the pages under `/learn`, so this pagination will cause some problem.

Reply:
And the path should not hard coded in the component. Instead, we need to read routes data from the manifest and render the pagination according to the routes in the manifest.